diff --git a/node_modules/asar/lib/asar.js b/node_modules/asar/lib/asar.js
index 8b98115..b5431b4 100644
--- a/node_modules/asar/lib/asar.js
+++ b/node_modules/asar/lib/asar.js
@@ -63,6 +63,11 @@ callback: The callback function. Accepts (err).
 module.exports.createPackageFromFiles = function (src, dest, filenames, metadata, options, callback) {
   if (typeof metadata === 'undefined' || metadata === null) { metadata = {} }
   if (typeof options === 'undefined' || options === null) { options = {} }
+
+  src = path.normalize(src)
+  dest = path.normalize(dest)
+  filenames = filenames.map(function (filename) { return path.normalize(filename) })
+
   const filesystem = new Filesystem(src)
   const files = []
   const unpackDirs = []