diff --git a/node_modules/patch-package/dist/makePatch.js b/node_modules/patch-package/dist/makePatch.js
index 6382c29..0a8a53d 100644
--- a/node_modules/patch-package/dist/makePatch.js
+++ b/node_modules/patch-package/dist/makePatch.js
@@ -58,6 +58,7 @@ exports.makePatch = function (packageName, appPath, packageManager, includePaths
         fs.writeFileSync(tmpRepoPackageJsonPath, JSON.stringify(deleteScripts(resolveRelativeFileDependencies_1.resolveRelativeFileDependenciesInPackageJson(appPath, require(path_1.join(tmpRepo.name, "package.json"))))));
         if (packageManager === "yarn") {
             fsExtra.copySync(path_1.join(appPath, "yarn.lock"), path_1.join(tmpRepo.name, "yarn.lock"));
+            fsExtra.copySync(path_1.join(appPath, ".yarnclean"), path_1.join(tmpRepo.name, ".yarnclean"));
             console.info(chalk_1.green("☑"), "Building clean node_modules with yarn");
             tmpExec("yarn");
         }
